### ISSUE TITLE
[BUG] Hands disappear on menu if you have servers.dat then delete servers.dat

### ISSUE THREAD

**Issue by KiwiFromDiscord** [QuestCraftPlusPlus/QuestCraft#138]
I may have found a bug that I believe I know the cause of.

I had the QuestCraft server file in my .minecraft folder, but I wanted to add Hypixel, so I attempted to manually add it into the .dat file.

I basically copied the line for the SMP but replaced it with Hypixels info and saved it again as a .dat .

I launched the game and clearly it didnt work as now no servers showed up.

But also my hands did not appear either, and only would appear if graphics were set to anything below Fabulous.

I then deleted the entire server.dat file and still, no arms on Fabulous graphics.

But, if I go into a world, my arms re appear.

Not gamebreaking in any way.

Also, the world at the menu screen for QC for some reason becomes daytime instead of night, for some reason, maybe that just happens.

TL;DR Steps to reproduce bug:

1. Add Server.dat file
2. Edit it with a new server and save as .dat
3. Delete it

**Comment by wellsilver** [comment]
I think it might be that you edited the server.dat wrong and it made the arms disappear because of spaghet code

**Comment by KiwiFromDiscord** [comment]
> I think it might be that you edited the server.dat wrong and it made the arms disappear because of spaghet code

Most likely, but I did delete the .dat file, and it was still happening. Atleast my arms are still there in the worlds,

**Comment by Mage-Enderman** [comment]
In the Main Title screen is it raining? if so it has probably nothing to do with your edits to the dat file
Rain causes hands to vanish afaik

**Comment by KiwiFromDiscord** [comment]
> 

Yes, I realized just there now that is the issue. For some reason last night I didnt even think about it.
